### DISCUSSION TITLE
feat(rate-limiting) Add a flag for not enforcing the rate-limit, just log

### DISCUSSION THREAD

**Discussion by nicholasklem** [Kong/kong#7337] [Ideas and feature requests]
Rate limiting often needs tuning after meeting production traffic.
Adjusting these can have unexpected consequences. I have a 
suggestion for a patch that  lets you run rate-limiting in dark mode, 
where it only logs what it would have done, without enforcing the 
limit.

Recommended in https://stripe.com/blog/rate-limiters

https://github.com/Kong/kong/pull/6834

**Comment by debu99** [comment]
currently it doesn't even log, it only return to the client...
